### PR TITLE
Mutualize docker image and integration tests for oracle11

### PR DESCRIPTION
**Pull request by simonvadee**
- Use base docker image with all db drivers (from arkhn/dockerfiles#5)
- Add the ORACLE11 database model
- Add integration tests for ORACLE11 (I found a docker image working out-of-the-box, youpi)

### CODE REVIEW

File: Dockerfile.test [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM arkhn/python-base:all
+FROM arkhn/python-base:test-all-drivers
```
Comment (Jasopaum) [inline Dockerfile.test:1]: Don't forget to change the tag

File: pagai/services/database_explorer.py [hunk 2]
```diff
@@ -36,11 +29,18 @@ def handle_between_filter(col, value):
 }
 
 MSSQL = "MSSQL"
+ORACLE11 = "ORACLE11"
```
Comment (Jasopaum) [inline pagai/services/database_explorer.py:32]: We'll need to enable ORACLE11 in pyrog, right?

**Review by Jasopaum — DISMISSED** [review]
LGTM

File: pagai/services/database_explorer.py (hunk 2 above)
Comment (simonvadee) [reply pagai/services/database_explorer.py:32]: indeed

File: Dockerfile.test (hunk 1 above)
Comment (simonvadee) [reply Dockerfile.test:1]: :+1:

**Review by Jasopaum — APPROVED** [review]

**Pull request merged**